app/controllers/steps/meursing_additional_codes_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Steps
+  class MeursingAdditionalCodesController < BaseController
+    def show
+      @step = Steps::MeursingAdditionalCode.new(meursing_additional_code_params)
+    end
+
+    def create
+      @step = Steps::MeursingAdditionalCode.new(meursing_additional_code_params)
+
+      validate(@step)
+    end
+
+    private
+
+    def meursing_additional_code_params
+      if params[:steps_meursing_additional_code].present?
+        params.require(:steps_meursing_additional_code).permit(:meursing_additional_code)
+      else
+        {}
+      end
```

Comment:
I think you can use fetch here, something like:
`params.fetch(foo:, {}).permit(:bar, :baz)`

(found on https://stackoverflow.com/questions/17687506/how-to-make-an-optional-strong-parameters-key-but-filter-nested-params)